protocol/src/types/block.rs

```diff
@@ -362,14 +362,14 @@ mod tests {
             }],
             consensus_config: ConsensusConfig {
                 gas_limit: 4294967295,
-                gas_price: 1,
                 interval: 3000,
                 propose_ratio: 15,
                 prevote_ratio: 10,
                 precommit_ratio: 10,
                 brake_ratio: 10,
                 tx_num_limit: 20000,
                 max_tx_size: 1024,
+                max_contract_limit: 0x6000
```

Comment:
Shall we add a test for the hardfork workflow into the multi-nodes job of axon-start-with-short-genesis.yml?

1. activate first hardfork
2. updateConsensusConfig